judge/admin/contest.py

```diff
@@ -233,19 +233,19 @@ def set_locked(self, request, queryset):
 
     def set_unlocked(self, request, queryset):
         for row in queryset:
-            self.set_is_locked(row, False)
+            self.set_locked_after(row, None)
         count = queryset.count()
         self.message_user(request, ungettext('%d contest successfully unlocked.',
                                              '%d contests successfully unlocked.',
                                              count) % count)
     set_unlocked.short_description = _('Unlock contest submissions')
 
-    def set_is_locked(self, contest, is_locked):
+    def set_locked_after(self, contest, locked_after):
         with transaction.atomic():
-            contest.is_locked = is_locked
+            contest.locked_after = locked_after
             contest.save()
             Submission.objects.filter(contest_object=contest,
-                                      contest__participation__virtual=0).update(is_locked=is_locked)
+                                      contest__participation__virtual=0).update(locked_after=locked_after)
```

Comment:
This will not affect submissions submitted after the lock time was set, but before the time of the lock.

Reply:
Fixed.